### PR TITLE
Cost Tab Tutorial

### PR DESCRIPTION
**Pull request by AndyHoNYC**
## Description

This adds a new workshop tutorial called Cost Tab that will tell users how to utilize the new Cost Tab feature on PCluster Manager

## Changes

Included new 07-cost-tab.md file
Included new image Activate-Tags-Button.png
Included new image Activate-Tags-Button.png
Included new image Budget-Name.png
Included new image Cost-Data.png
Included new image Create-Budget.png
Included new image Custom-Range.png
Included new image Date-Filter.png
Included new image Progress-Bar.png

## How Has This Been Tested?

Tested Locally 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/02-tutorials/07-cost-tab.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
++++
+title = "f. Cost Tab 📈"
+weight = 27
++++
+
+This tutorial tells you how to view cluster spending costs and created budgets for clusters
+
+## Setup
+
+1. Cost Allocation Tags allow us to track spending of parallelclusters from parallelcluster specific tags, so first we must activate those tags.
```
Comment (sean-smith) [inline docs/content/02-tutorials/07-cost-tab.md:11]: Let's make this the introduction, not a numbered step.

File: docs/content/02-tutorials/07-cost-tab.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+
++++
+title = "f. Cost Tab 📈"
+weight = 27
++++
+
+This tutorial tells you how to view cluster spending costs and created budgets for clusters
```
Comment (sean-smith) [inline docs/content/02-tutorials/07-cost-tab.md:7]: "This tutorial **shows** you how to view cluster costs broken down by instance type and create a budget for a specific cluster to get alerted when you approach your budget's cap."

File: docs/content/02-tutorials/07-cost-tab.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+## Setup
+
+1. Cost Allocation Tags allow us to track spending of parallelclusters from parallelcluster specific tags, so first we must activate those tags.
+2. Press the "Activate Tags" button to activate Cost Allocation Tags
```
Comment (sean-smith) [inline docs/content/02-tutorials/07-cost-tab.md:12]: "Click the.."

File: docs/content/02-tutorials/07-cost-tab.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+![Cost Tags](cost-tab/Activate-Tags-Button.png)
+
+3. If cost allocation tags were already activated, then the Activate Tags Button should look "grayed out"
```
Comment (sean-smith) [inline docs/content/02-tutorials/07-cost-tab.md:15]: "grayed out" shouldn't have quotes around it

File: docs/content/02-tutorials/07-cost-tab.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+![Cost Tags](cost-tab/Activated_Button.png)
+
+## Filter Data by dates
+
+Now you can check spending costs by cluster based on range of specified dates
```
Comment (sean-smith) [inline docs/content/02-tutorials/07-cost-tab.md:20]: > Now you can check spending costs by cluster based on range of specified dates

Should become "Now you can check spending costs by cluster based and filter by time range."

File: docs/content/02-tutorials/07-cost-tab.md (hunk 1 above)
Comment (AndyHoNYC) [reply docs/content/02-tutorials/07-cost-tab.md:11]: Resolved

File: docs/content/02-tutorials/07-cost-tab.md (hunk 2 above)
Comment (AndyHoNYC) [reply docs/content/02-tutorials/07-cost-tab.md:7]: Resolved

File: docs/content/02-tutorials/07-cost-tab.md (hunk 3 above)
Comment (AndyHoNYC) [reply docs/content/02-tutorials/07-cost-tab.md:12]: Resolved

File: docs/content/02-tutorials/07-cost-tab.md (hunk 4 above)
Comment (AndyHoNYC) [reply docs/content/02-tutorials/07-cost-tab.md:15]: Resolved

File: docs/content/02-tutorials/07-cost-tab.md (hunk 5 above)
Comment (AndyHoNYC) [reply docs/content/02-tutorials/07-cost-tab.md:20]: Resolved

**Review by AndyHoNYC — COMMENTED** [review]
Resolved Changes

**Pull request merged**